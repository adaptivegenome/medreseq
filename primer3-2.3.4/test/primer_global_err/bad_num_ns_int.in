SEQUENCE_ID=bad_num_ns_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_NS_ACCEPTED=-1
=
