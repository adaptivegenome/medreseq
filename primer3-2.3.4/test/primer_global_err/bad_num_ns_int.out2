../src/primer3_core: Illegal value for PRIMER_INTERNAL_MAX_NS_ACCEPTED
