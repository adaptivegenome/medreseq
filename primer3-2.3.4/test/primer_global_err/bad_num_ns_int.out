SEQUENCE_ID=bad_num_ns_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_MAX_NS_ACCEPTED=-1
PRIMER_ERROR=Illegal value for PRIMER_INTERNAL_MAX_NS_ACCEPTED
=
